docs/cli/configuration.md

```diff
@@ -179,13 +179,24 @@ In addition to a project settings file, a project's `.gemini` directory can cont
     }
     ```
 - **`usageStatisticsEnabled`** (boolean):
+
```

Comment:
Can you leave the whitespace alone for the other entries in this file or make them all consistent? Seems like the file still has a mix of the two options.